tldw_Server_API/app/core/MCP_unified/tests/test_flashcards_module.py

```
@@ -102,7 +102,7 @@ async def test_flashcards_crud_and_export(monkeypatch):
     fake_db = FakeFlashcardsDB()
     mod._open_db = lambda ctx: fake_db  # type: ignore[attr-defined]
 
-    ctx = SimpleNamespace(db_paths={"chacha": "/tmp/chacha.db"})
+    ctx = SimpleNamespace(db_paths={"chacha": "/tmp/chacha.db"})  # nosec B108
```

Ruff (0.15.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/chacha.db"

(S108)